x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/rules_table/rules_table_context.tsx

```diff
@@ -196,10 +201,12 @@ export const RulesTableContextProvider = ({ children }: RulesTableContextProvide
 
   const isActionInProgress = useMemo(() => {
     if (loadingRules.ids.length > 0) {
-      return !['disable', 'enable', 'edit'].includes(loadingRules.action ?? '');
+      return (
+        loadingRules.isDryRun || !['disable', 'enable', 'edit'].includes(loadingRules.action ?? '')
+      );
```

Comment:
Instead of adding a new `isDryRun` prop, can we return `true` when any bulk action is in progress? I.e., remove the `'disable', 'enable', and 'edit'` checks as well.